geode-core/src/main/java/org/apache/geode/pdx/internal/PeerTypeRegistration.java

```diff
@@ -127,6 +127,13 @@ public PeerTypeRegistration(InternalCache cache) {
 
   @Override
   public void initialize() {
+    // Relevant during reconnect
+    TypeRegistry typeRegistry = cache.getPdxRegistry();
+    if (typeRegistry != null) {
+      typeRegistry.flushCache();
+      logger.info("Flushing TypeRegistry");
```

Comment:
I was wondering why this is an info message. Will it really mean anything to end users? Maybe debug would be better than info.

Reply:
Fixed